.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,51 @@
+<!--
+PLEASE HELP US PROCESS GITHUB ISSUES FASTER BY PROVIDING THE FOLLOWING INFORMATION.
+ISSUES MISSING IMPORTANT INFORMATION MAY BE CLOSED WITHOUT INVESTIGATION.
+-->
+
+## I'm submitting a...
+<!-- Please uncomment one or more that apply to this issue -->
+
+<!-- - Regression (a behavior that used to work and stopped working in a new release) -->
+<!-- - Bug report (I searched for similar issues and did not find one) -->  
+<!-- - Feature request --> 
+<!-- - Sample app request -->
+<!-- - Documentation issue or request -->
+<!-- - Question of Support request => Please do not submit support request here, instead see https://github.com/nventive/Uno/blob/master/README.md#have-questions-feature-requests-issues -->
```

Comment:
the link points to uno, do we have an equivalent in uno.core?

Reply:
I will check